src/review09_10/Puzzle09.java

```diff
@@ -0,0 +1,25 @@
+package review09_10;
+
+import java.util.*;
+
+public class Puzzle09 {
+
+    // With what do we need to replace the `null` to get the output [5, 4, 3, 2, 1] ? Choose all that apply
+    //
+    // A. new TreeSet<>((a, b) -> b - a)
+    // B. new TreeSet<>((a, b) -> a - b)
+    // C. new PriorityQueue<>((a, b) -> b - a)
```

Comment:
Komt de interne volgorde van PriorityQueue uberhaupt aan de orde in de scope van OCP?

Reply:
Nee, volgens mij komt dat niet aan de orde. Alleen weten dat een PrioQueue bestaat en een implementatie van Deque is